framework/tester.py

```diff
@@ -328,3 +328,15 @@ def wait_while_busy(self, *items):
         for item in items:
             if item.is_running():
                 item.wait_for_finish()
+
+    # Should replace all duplicated instances of wait_all_connections
+    def wait_all_connections(self, tmt=1):
+        for sid in self.__servers:
+            srv = self.__servers[sid]
+            if not srv.wait_for_connections(timeout=tmt):
+                return False
+        return True
+
+class SingleTest(object):
+    def test(self):
+        self._test()
```

Comment:
LF is required

Reply:
It seems this class is introduced for the only goal not to write
```C
   def test(self):
        self._test()
```
in every class in `test_cache_control.py`.

It seems, the class should be a litle more documented.

Also probably it should be placed in `test_cache_control.py` itself rather than in `framework/tester.py` ?